### PR TITLE
Tuphan

### PR DESCRIPTION
**Pull request by phanhungtu**
Please pull code & merge to master

### CODE REVIEW

File: config/sync/user.role.club_admin.yml [hunk 1]
```diff
@@ -6,7 +6,8 @@ id: club_admin
 label: 'Club Admin'
 weight: 6
 is_admin: null
-permissions:
+permissions:  
+  - 'administer users'
```
Comment (dgusick) [inline config/sync/user.role.club_admin.yml]: Why do u need this permission?

File: web/sites/default/settings.php [hunk 2]
```diff
@@ -782,11 +782,10 @@
  * Keep this code block at the end of this file to take full effect.
  */
 #
-//$config['system.logging']['error_level'] = 'verbose';
 $databases['default']['default'] = array (
-  'database' => 'cmsgit',
+  'database' => 'cms',
```
Comment (dgusick) [inline web/sites/default/settings.php]: don't push database config changes

File: config/sync/user.role.club_admin.yml (hunk 1 above)
Comment (phanhungtu) [reply config/sync/user.role.club_admin.yml]: This permission for user assign to team page

File: web/sites/default/settings.php (hunk 2 above)
Comment (phanhungtu) [reply web/sites/default/settings.php]: I don't commit any from this file, it's merged from master branch

**Pull request merged**